packaging/fix_conda_dylib_paths.sh

```diff
@@ -0,0 +1,33 @@
+function fix_velox_dylib_paths() {
+  velox_so=$1
```

Comment:
nit: can we check there is exactly one parameter ?

Reply:
can you elaborate?